app/javascript/githubRepos/singleRepo.jsx

```diff
@@ -0,0 +1,75 @@
+import { h, Component } from 'preact';
+import PropTypes from 'prop-types';
+
+export class SingleRepo extends Component {
+  constructor(props) {
+    super(props);
+    const { selected } = this.props;
+    this.state = { selected };
```

Comment:
I originally had `this.state = this.props.selected` but the linter threw an error saying I needed to use destructuring assignment. I see that it's useful but it seems more verbose to me (literally two lines versus one). Also, I had to do change it everywhere else where I was using `this.props.something`, so personally think we should change that rule.